UnrealPlugin/README.md

```diff
@@ -10,337 +11,38 @@ It was tested with Unreal Engine 4.26.2, but should work with other engines as w
+* [Optional] [PlayFab Marketplace plugin](https://www.unrealengine.com/marketplace/product/playfab-sdk) or the [source version on GitHub](https://github.com/PlayFab/UnrealMarketplacePlugin/tree/master/4.26/PlayFabPlugin/PlayFab) (Not required for GSDK, but required for most other PlayFab services, including login)
 
-Then open the .uproject file.
+## Project creation
 
-In the top left of the Unreal editor menu, click on File -> Package -> Target Configuration -> [game name]Server and then go to File -> Package -> Windowsx64. 
-You can now use this packaged version of your game server to [test with LocalMultiplayerAgent](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab) 
-or to use it directly with PlayFab by [creating a build](https://developer.playfab.com/) and then using [PlayFab MpsAllocatorSample](https://github.com/PlayFab/MpsSamples/blob/master/MpsAllocatorSample/README.md).
+Unreal GSDK is installed into an Unreal server project. You will need a network-enabled multiplayer Unreal project with a dedicated-server mode. If you do not have one, you can follow our [Unreal prerequsitie setup](ThirdPersonMPSetup.md) guide. Once you have a network-enabled, multiplayer game, with a dedicated server, return here and continue.
 
-## Setting up a Windows Dedicated Server on PlayFab
-An important note about this is that you need to set the start command in process-based mode to \<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
+## Project GSDK setup
 
-In container-based mode use \<mount folder\>\\\<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
+With a multiplayer project configured for both a network connected game-client and game-server, you're ready to [install the GSDK](ThirdPersonMPGSDKSetup.md). This will allow your server to run on PlayFab Multiplayer Services.
 
-**If you use the executable in the root folder, the server will fail to initialize with the PlayFab system.**
+## Unreal Project Build Configurations
 
-## Setting up a Linux Dedicated Server on PlayFab
-During testing the following Dockerfile + startup.sh script worked excellent with the PlayFab Linux VM:
+Once the GSDK is installed and configured in your project, you can [build your Unreal project](ThirdPersonMPBuild.md). There are multiple build configurations you will need to execute the full range of tests in future steps.
 
-### The Dockerfile:
-```Dockerfile
-FROM ubuntu:18.04
+## Test Local Server Deployment
 
-# Unreal refuses to run as root user, so we must create a user to run as
-# Docker uses root by default
-RUN useradd --system ue
-USER ue
+Once you have any client build, and a "Development Server" build, you can [test the MPS deployment process locally](ThirdPersonMPLocalDeploy.md), and verify your server works.
```

Comment:
Once you have a client build and a PlayFab Multiplayer Game Server Build, [test the MPS deployment process locally](ThirdPersonMPLocalDeploy.md) and verify your server works. For more information about PlayFab Multiplayer Game Server Build, see [Terminology] (https://docs.microsoft.com/gaming/playfab/features/multiplayer/servers/server-terms)

Reply:
In this case, I'm using a term that is specific to Unreal, and does not come from the terms page you linked.
"Development Server" is the configuration setting set in Unreal to build this specific Unreal game server. It is described in the ThirdPersonMPBuild.md document under that name, and these instructions specifically require that build configuration.
I'm going to keep my own version of this line, but let me know if you have any additional comments.